setup.py

```diff
@@ -26,8 +26,7 @@
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3 :: Only',
```

Comment:
Didn't know about that `:: Only` classifier, TIL!

Reply:
Angry people tell me about these things when I forget to use them... 😀